docs/notebooks/ztf_bts-ngc.ipynb

```diff
@@ -0,0 +1,197 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    from astroquery.vizier import Vizier
</pre><p>Can we instead just pip install directly from the notebook, instead of adding as a dev dependency? We do this in the import catalogs notebook like:</p><p> </p><pre class="ql-syntax" spellcheck="false">!pip install &lt;package&gt; --quiet
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/142/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Done</p>